server/form-pages/apply/location-factors/pduTransfer.ts

```diff
@@ -0,0 +1,54 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { Application } from '../../../@types/shared'
+
+import TasklistPage from '../../tasklistPage'
+
+type Response = 'yes' | 'noNeedToMakeArrangements' | 'noProbationPractitioner'
```

Comment:
To be more DRY and to match how we've done it elsewhere, it might be worth structuring the questions and responses like so:

```suggestion
const questions = {
    yes: 'Yes',
    noNeedToMakeArrangements: 'No, I still need to make arrangements',
    noProbationPractitioner: 'No, management and supervision will be maintained by the existing Probation Practitioner',
} as const

const responses = Object.keys(questions)

type Response = keyof typeof responses
```